auto/src/glew_init_gl.c

```diff
@@ -5,8 +5,22 @@ GLboolean GLEWAPIENTRY glewGetExtension (const char* name)
   const GLubyte* start;
   const GLubyte* end;
   start = (const GLubyte*)glGetString(GL_EXTENSIONS);
```

Comment:
glGetString(GL_EXTENSIONS) will generate an error and return NULL on a core context. The code will then try to use GL_NUM_EXTENSIONS. While this causes an error on core context I did it this way because I can be sure it will not break compatibility contexts. I think (but haven't checked) trying to get GL_NUM_EXTENSIONS on a pre-3.0 context will generate an error and so would change previously-working code to working-but-raises-an-error.

There are better ways to handle this. We could first check if we have >3.0 context but this was the least invasive way I could do it and would have required more testing than I was willing to do at that point.